src/express/middlewares.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash-firecloud';
+import reqMixins from './req-mixins';
+import resMixins from './res-mixins';
+
+import {
+  asyncHandler,
+  getRequestInstance
+} from '../lambda';
+
+import {
+  bootstrap as bootstrapResponseError
+} from './res-error';
+
+export let bootstrap = function(fn) {
+  return asyncHandler(async function(req, res, next) {
+    bootstrapResponseError(await _.alwaysPromise(fn(req, res, next)), res);
+  });
+};
+
+export let init = function(req, res, next) {
+  res.instance = getRequestInstance(req);
```

Comment:
I understand that this is not part of this PR, but this line is misleading if you are out of context. We assign a property `instance` for the response which is an instance of `request` according to the right-hand value 🤔 
But in fact it's much easier: that's just a request id from AWS. So why not calling things what they are?